src/client/kinto_http.rs

```diff
@@ -68,10 +79,19 @@ pub async fn get_records(
 
     let size = resp.headers().get("content-length").ok_or_else(|| -1);
     debug!("Download {:?} bytes...", size);
+    
     let body = resp.text().await?;
-    let result: ChangesetResponse = serde_json::from_str(&body)?;
 
-    Ok(result)
+    let latest_change: KintoPluralResponse<LatestChangeResponse> = serde_json::from_str(&body)?;
+    
+    let last_modified = match latest_change.data.get(0) {
+        Some(change) => change.last_modified,
+        _ => 0 // 0 by default
```

Comment:
We should raise an error here. If there is no entry in this endpoint, it means the bucket/collection is unknown. The call to changeset will raise a 404, so we better fail early

Reply:
done